Home_work1.py

```diff
@@ -0,0 +1,24 @@
+# Декілька способів, можно ще модулі підв'язати, але це буде окремий файл
+print ('Hello world')
+text_1 = 'Hello world' 
+text_2 = 'again'
+print (text_1, text_2, sep=',', end='!\n')
+print ('{0},{1}' .format(text_1, text_2), end='!\n')
+print (f"{text_1}, {text_2}!\n\n")
+

```

Comment:
Complete task 2, where you put expected result in comment and create this result using features of print after comment

Reply:
Done